src/foundations/shape/Shape.stories.mdx

```diff
@@ -39,20 +36,9 @@ export const argTypes = {
             type: 'select',
         },
     },
-    elevation: {
-        options: SHAPE_ELEVATION_LEVELS,
-        control: {
-            type: 'select',
-        },
-    },
-    elevationOnHover: {
-        options: SHAPE_ELEVATION_LEVELS,
-        control: {
-            type: 'select',
-        },
-    },
     width: { control: { type: 'range', min: 150, max: 600, step: 25 } },
     height: { control: { type: 'range', min: 150, max: 600, step: 25 } },
+    ...Utils.hideComponentProperties(['elevation', 'elevationOnHover']),
```

Comment:
nit: shouldn't we still have the option to set elevations based on props on the `Shape` component?